### PR TITLE
Team stats

### PR DESCRIPTION
**Pull request by araspaas**
Just got 6 outta the 8 methods done and working

### CODE REVIEW

File: lib/team_stats.rb [hunk 1]
```diff
@@ -0,0 +1,62 @@
+class TeamStats < Stats
```
Comment (ShaunDaneJames) [inline lib/team_stats.rb:1]: I like how the inheritance makes the file much cleaner.

**Review by ShaunDaneJames — APPROVED** [review]
Looks like some of the variables could be broken down into helper methods.

File: lib/team.rb [hunk 2]
```diff
@@ -4,7 +4,7 @@ class Team
 
   def initialize(team_param)
     @team_id = team_param[:team_id]
-    @franchise_id = team_param[:franchise_id]
+    @franchise_id = team_param[:franchiseid]
```
Comment (dcoleman21) [inline lib/team.rb:7]: good catch on this

**Review by robertorodriguez12 — COMMENTED** [review]
I am loving the simplicity of this method. I think it looks great! Thanks for all of your hard work Austin.

**Review by dcoleman21 — COMMENTED** [review]
These look great! Let's see how we can refactor some of these.

**Review by robertorodriguez12 — APPROVED** [review]

**Pull request merged**